src/experiments/OOD-adj-scores/models/.gitkeep

```diff
@@ -0,0 +1 @@
+# trained models go in this folder
```

Comment:
Do you think it is better to distribute this folder, or just call `!mkdir models` inside of the notebook before saving the trained models?

Reply:
Switched to calling `!mkdir models` instead.